records_mover/db/bigquery/bigquery_db_driver.py

```diff
@@ -110,6 +116,17 @@ def tweak_records_schema_for_load(self,
             #
             # So we need to make sure we don't create any DATETIME
             # columns if we're loading from a Parquet file.
+            #
+            # https://cloud.google.com/bigquery/docs/loading-data-cloud-storage-parquet
             return records_schema.convert_datetimes_to_datetimetz()
+        elif isinstance(records_format, AvroRecordsFormat):
+            # https://cloud.google.com/bigquery/docs/loading-data-cloud-storage-avro#logical_types
+            #
+            # "Note: There is no logical type that directly
+            # corresponds to DATETIME, and BigQuery currently doesn't
+            # support any direct conversion from an Avro type into a
+            # DATETIME field."
+            #
+            return records_schema.convert_datetimes_to_string()
```

Comment:
This came up during the table2table integration test; BigQuery fails to load if you try to load the same string it exports from a DATETIME column into a DATETIME column.

Reply:
😬